improver_tests/spotdata/spotdata/test_SpotExtraction.py

```diff
@@ -74,19 +74,25 @@ def setUp(self):
         # can be tested
         attributes = {"mosg__grid_domain": "global", "mosg__grid_type": "standard"}
 
+        cell_methods = (
+            iris.coords.CellMethod("maximum", coords="time", intervals="1 hour"),
+        )
+
```

Comment:
Make this self.cell_methods, thus removing the need to redefine it in the test below.

Reply:
I've updated this as suggested.